src/lib/ProtocolAdapterFactory.ts

```diff
@@ -30,8 +30,13 @@ export const ProtocolAdapterFactory = (() => {
     return protocolAdapterRegistry[protocol]!;
   }
 
+  function getAllAdapters(): ProtocolRecord<BaseProtocolAdapter> {
+    return protocolAdapterRegistry;
+  }
```

Comment:
I don't think it's reasonable to create this method. If you want to loop the protocols and find something I suggest for example:
```
PROTOCOLS.find((protocol) => {
  const adapter = ProtocolAdapterFactory.getAdapter(protocol);
  return adapter.<UseSomeAdapterMethodHere>() == <ComparedValue>;
});
```

Reply:
You mean to define a method in this file to get one property of each protocol?
For example, `getAllProtocolSymbols` ?

Reply:
Fixed